python/samples/getting_started/azure_functions/05_multi_agent_orchestration_concurrency/function_app.py

```diff
@@ -63,14 +64,18 @@ def multi_agent_concurrent_orchestration(context: DurableOrchestrationContext):
     physicist_thread = physicist.get_new_thread()
     chemist_thread = chemist.get_new_thread()
 
-    physicist_task = physicist.run(messages=str(prompt), thread=physicist_thread)
-    chemist_task = chemist.run(messages=str(prompt), thread=chemist_thread)
+    # Create generators from agent.run() calls
+    physicist_gen = physicist.run(messages=str(prompt), thread=physicist_thread)
+    chemist_gen = chemist.run(messages=str(prompt), thread=chemist_thread)
```

Comment:
I think we should be able to revert these changes now that we're returning tasks again.